app/src/main/java/fr/free/nrw/commons/CommonsApplication.java

```diff
@@ -135,7 +136,7 @@ public AppLanguageLookUpTable getLanguageLookUpTable() {
     ContributionDao contributionDao;
 
     /**
-     * In memory list of contributios whose uploads ahve been paused by the user
+     *  In memory list of contribution whose uploads have been paused by the user
```

Comment:
Note for the future: Thanks for fixing these! Please do not hesitate to make such changes in a separate pull request, so that this pull request does not contain unrelated changes :-)

By the way:
In memory -> In-memory
contribution -> contributions